lib/Basics/files.cpp

```diff
@@ -2092,7 +2092,11 @@ int TRI_Crc32File(char const* path, uint32_t* crc) {
 
     if (sizeRead < bufferSize) {
       if (feof(fin) == 0) {
-        res = errno;
+        if (errno != 0) {
+          res = TRI_set_errno(TRI_ERROR_SYS_ERROR);
+        } else {
+          res = TRI_ERROR_NO_ERROR;
+        }
```

Comment:
@jsteemann fyi, I'm gonna change this again because it seems to be nonsense: `feof` doesn't set `errno` at all, and no system or library function sets `errno` to `0`.
```suggestion
        res = TRI_ERROR_INTERNAL;
```
Do you think we need a more specific error code? It would probably have to be introduced, or we can change the return code to something else (e.g. `ResultT<bool>` or so).

Reply:
`TRI_ERROR_FAILED` is fine for me too.